File: RR-Frontend/src/app/core/models/shared/login/login-request.ts
export class LoginRequest {
  username: string;
  password: string;
  instituteId: string;
  application: string;
  rememberMe: boolean;

  constructor(username: string, password: string, instituteId: string, rememberMe: boolean) {
    this.username = username;
    this.password = password;
    this.instituteId = instituteId;
    this.application = 'RR';
    this.rememberMe = rememberMe;
  }
}

File: RR-Backend/RadioReport.Core/Users/RadioReport.Core.Users.Domain/Models/LoginRequest.cs
using System;
using System.ComponentModel.DataAnnotations;
using RadioReport.Common.Logic.Enums;

namespace RadioReport.Core.Users.Domain.Models
{
    public class LoginRequest
    {
        [Required]
        public string UserName { get; set; }

        [Required]
        public string Password { get; set; }

        [Required]
        public Application Application { get; set; }
        
        [Required]
        public Guid InstituteId { get; set; }

        public bool RememberMe { get; set; }
    }
}
